### ISSUE TITLE
Document Why Chromeview > WebView

### ISSUE THREAD

**Issue by JohnMcLear** [thedracle/cordova-android-chromeview#9]
I assume the main point of this repository is to persuade me to use chromeview over webview.  The readme should probably document the advantages of Chromeview over Webview.  I naively assume it's support for WebRTC/WebGL and support for other standards that are hitting the chromium releases before they are hitting Webview.

Some references to https://github.com/pwnall/chromeview would probably be beneficial

**Comment by thedracle** [comment]
https://developers.google.com/chrome/mobile/docs/webview/overview

**Comment by thedracle** [comment]
Mainly:
"Does the new WebView have feature parity with Chrome for Android?

For the most part, features that work in Chrome for Android should work in the new WebView.

Chrome for Android supports a few features which aren't enabled in the WebView, including:

WebGL 3D canvas
WebRTC
WebAudio
Fullscreen API
Form validation"

And this is even with WebView now being based on Chromium.

This is maybe a way for people to get a preview of how webview will work in the future. Or potentially make more modern HTML5 web applications.

**Comment by JohnMcLear** [comment]
TLDR; 
- WebGL 3D canvas
- WebRTC
- WebAudio
- Form validation

Fullscreen API being not worth mentioning as it's already available
